### ISSUE TITLE
Environment.yml

### ISSUE THREAD

**Issue by CommonClimate** [LinkedEarth/FreqFun#12]
We need to update the environment with all the packages that were used to run the NBs. This will involve the usual stack (Pyleoclim, Xarray, and dependencies), as well as whatever package is introduced in each notebook. This will be essential for #11

**Comment by jordanplanders** [comment]
In the current environment.yml file, Pyleoclim is pinned at 0.7.4. Do we want to change that?

My most recent notebook runs safely (with `@matplotlib notebook` instead of `@matplotlib ipympl` because we are currently running Jupyter notebook rather than Jupyter lab) once I install `aiohttp`, `zarr`, `nc-time-axis`. For what it's worth, it also automatically downloads coastlines from https://naturalearth.s3.amazonaws.com/110m_physical/ne_110m_coastline.zip, and while I'm not sure how to cache them, it seems like that would be an efficient step to take.

**Comment by CommonClimate** [comment]
Let's discuss this on Thursday at the PaleoCube meeting, re: "several repositories with different “themed” notebooks"

**Comment by CommonClimate** [comment]
In general, we should strive to align this repo to the latest Pyleoclim release, whenever possible. It might no longer be possible when we have tons of NBs, and we should allow people to submit those workflows as "science snapshots" tied to a moment in time (and release).

**Comment by CommonClimate** [comment]
@jordanplanders are we ready to close this issue?

**Comment by jordanplanders** [comment]
@CommonClimate This is also related to #14. 

The widget notebook is part of the LMR-CMIP6 paleobook, so we don't need to support it (and the widget idiosyncrasies) here. I like the idea of a core minimal environment and we can specify in-notebook installs as needed.

**Comment by CommonClimate** [comment]
Yes. Wouldn't you say that the pyleoclim tutorials `environment.yml` is that minimal happy place? If so, let's make a note of it and close this.

**Comment by jordanplanders** [comment]
At the moment, you use pangaeapy and climlab, and I think an environment file that gets both of these happily installed with Pyleoclim is a good idea. When I installed pangaeapy after my initial environment setup I had to reinstall the correct version of numpy to avoid conflicts, so I think this is simpler way to handle things.

**Comment by khider** [comment]
Addressed in https://github.com/LinkedEarth/FreqFun/commit/68056a7fb21fb27d819818f9f4de9989a0161bd8
